tachikoma-it/src/test/kotlin/com/sourceforgery/tachikoma/database/dao/BlockedEmailDAOSpec.kt

```diff
@@ -2,13 +2,14 @@ package com.sourceforgery.tachikoma.database.dao
 
```

Comment:
BlockedEmailDAOTest not Spek

Reply:
The same goes for all old Spek files

Reply:
Right, forgot to rename. I was going to do that in a different commit.